pages/c/_cluster/apps/charts/index.vue

```diff
@@ -17,10 +19,50 @@ import { removeObject, addObject, findBy } from '@/utils/array';
 import { compatibleVersionsFor, filterAndArrangeCharts } from '@/store/catalog';
 import { CATALOG } from '@/config/labels-annotations';
 
+const sliderData = [
+  {
+    id:      '1',
+    img:     'img1.jpg',
+    partner: 'Partner',
```

Comment:
What does this do? this is not in use right?

Reply:
This is to create a badge
<img width="234" alt="Screenshot 2022-04-25 at 20 16 01" src="https://user-images.githubusercontent.com/18264463/165113946-a8a45d92-8410-470d-85a2-bec2a7482f54.png">